### PR TITLE
1.0.3 version

### PR DESCRIPTION
**Pull request by ghost**
**Note**: available at `[org.clojars.tristefigure/reader-macros "1.0.2.1"]`.

This PR bumps the version tag in `project.clj` and in the `README` (the last PR (https://github.com/klutometis/reader-macros/pull/6) was not released on Clojars, see issue https://github.com/klutometis/reader-macros/issues/5).

@klutometis : I just remembered you added me as collaborator a few months ago but I'll let you merge this one since I don't have the rights to release on the `reader-macros` on Clojars (https://clojars.org/groups/reader-macros will let you do this if you wish so).

### CODE REVIEW

File: README.org [hunk 1]
```diff
@@ -122,7 +122,7 @@
   Add the following to your dependencies in =project.clj=:
 
   #+BEGIN_SRC clojure
-    [reader-macros "1.0.2"]
+    [reader-macros "1.0.3"]
```
Comment (TristeFigure) [inline README.org:125]: here.
My editor cleaned all the empty lines  :^o

**Comment by klutometis** [comment]
Thanks, @TristeFigure; I'll add you on Clojars: same username?

**Comment by klutometis** [comment]
Added!

**Comment by ghost** [comment]
Excellent ! The latest release is now available on Clojars via `[reader-macros "1.0.3"]`.

**Pull request merged**